contracts/interfaces/IDecentAutonomousAdmin.sol

```diff
@@ -0,0 +1,21 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.28;
+
+import {IHats} from "./hats/full/IHats.sol";
+import {ISablierV2Lockup} from "./sablier/full/ISablierV2Lockup.sol";
+
+interface IDecentAutonomousAdmin {
+    struct SablierStreamInfo {
+        uint256 streamId;
+        ISablierV2Lockup sablierV2Lockup;
+    }
```

Comment:
I think this struct (and the `ISablierV2Lockup`) are unused and can be removed

Reply:
🤦 oof yeah good call